lib/ruby-osc/version.rb

```diff
@@ -1,4 +1,4 @@
 # encoding: UTF-8
 module Osc
-  VERSION = "0.4.1"
+  VERSION = "0.40.0"
```

Comment:
@nicb I would say bump would be to v `0.4.1` since actually just fixes a deprecation warning and there is no change in behavior

While at it could you change string literal to single quotes to satisfy houndci?